src/ShiftedProximalOperators.jl

```diff
@@ -67,6 +71,13 @@ function shift!(ψ::ShiftedProximableFunction, shift::AbstractVector{R}) where {
   return ψ
 end
 
+function shift!(ψ::ShiftedCompositeProximableFunction, shift::AbstractVector{R}) where {R <: Real}
+  ψ.c!(shift,ψ.b)
+  ψ.J!(shift,ψ.A)
+  ψ.is_shifted = true
+  return ψ
+end
+
```

Comment:
We start from $h(x) = g(c(x))$. What is the type of $h$? It hasn't been shifted yet. It should be something like a `CompositeProximableFunction`, and, when we call `shift!(h, s)`, we get a `ShiftedCompositeProximableFunction` as output.